modules/custom/az_person/config/install/views.view.az_reorder.yml

```diff
@@ -445,6 +445,7 @@ display:
           empty_table: false
           caption: ''
           description: ''
+          class: ''
```

Comment:
See https://git.drupalcode.org/project/drupal/-/blob/11.x/core/modules/views/config/schema/views.style.schema.yml?ref_type=heads#L127

See https://www.drupal.org/project/drupal/issues/3045871

Reply:
We do appear to be missing the sticky config .